### ISSUE TITLE
SkeletalMeshDrawRadius

### ISSUE THREAD

**Issue by postm1** [kirides/GD3D11#163]
SkeletalMeshDrawRadius has a constant value of 6000. Some users play with bigger distance setting and npcs are invisible.

**Suggestion:**
Add into the setting file SkeletalMeshDrawRadius and read it in **LoadMenuSettings**

 s.SkeletalMeshDrawRadius = GetPrivateProfileIntA( "General", "SkeletalMeshDrawRadius", 6000, ini.c_str() );

Or even better: create a scrollbar with this setting ;)

**Comment by kirides** [comment]
is this still needed?

**Comment by postm1** [comment]
Nope, this is already in f11 menu (King's version). But you have ur new menu ;)

**Comment by kirides** [comment]
> Nope, this is already in f11 menu (King's version). But you have ur new menu ;)

@postm1 yes, i decided that using ImGui as the other fork did, was the right direction.
this allowed to completely ditch D2D1 old settings and AntTweakBar, thus reducing the dependencies quite a bit.
and ImGui is much more fleshed out than the D2d1 UI of the old renderer

i will also add this setting to persistence then, thanks :)
